SpatialGDK/Source/SpatialGDK/Public/Schema/ComponentPresence.h

```diff
@@ -43,10 +44,10 @@ struct ComponentPresence : Component
 		Data.schema_type = Schema_CreateComponentData();
 		Schema_Object* ComponentObject = Schema_GetComponentDataFields(Data.schema_type);
 
-		for (const Worker_ComponentId& InComponentId : ComponentList)
-		{
-			Schema_AddUint32(ComponentObject, SpatialConstants::COMPONENT_PRESENCE_COMPONENT_LIST_ID, InComponentId);
-		}
+		uint32 BufferSize = ComponentList.Num();
+		uint32* Buffer = reinterpret_cast<uint32*>(Schema_AllocateBuffer(ComponentObject, BufferSize));
```

Comment:
```suggestion
		uint32* Buffer = reinterpret_cast<uint32*>(Schema_AllocateBuffer(ComponentObject, BufferSize * sizeof(utin32)));
```
I think this needs to be bytes rather than elements.

Reply:
Yep 🤦